frontend/src/models/models.js

```diff
@@ -208,6 +208,10 @@ module.exports = app => app.component('models', {
       this.filteredPaths = [...this.schemaPaths];
       this.selectedPaths = [...this.schemaPaths];
     },
+    deselectAll() {
+      this.selectedPaths.length = 0;
+      this.filteredPaths.length = 0;
```

Comment:
This also hides all documents before the user clicks "Filter selection". Before:

![image](https://github.com/mongoosejs/studio/assets/1620265/ac09a4ed-419f-40bc-a553-4383720997fe)

After click:

![image](https://github.com/mongoosejs/studio/assets/1620265/5f810f63-10c9-4fee-9d79-a0cba8cc17ba)